src/box/lua/load_cfg.lua

```diff
@@ -148,6 +154,7 @@ local default_cfg = {
     wal_dir_rescan_delay= 2,
     wal_queue_max_size  = 16 * 1024 * 1024,
     wal_cleanup_delay   = 4 * 3600,
+    wal_retention_delay = ifdef_wal_retention_delay(0),
```

Comment:
I find the option name rather unfortunate. In English it means "delay of retention" but actually it denotes the retention time interval so I'd call it `wal_retention_time` or `wal_retention_interval` or `wal_retention_period`. It might be wroth consulting the doc team regarding the option name.

Reply:
@tarantool/doc, what do you think of `wal_retention_delay` name for an option, which  postpones the deletion of wal files for a number of seconds?

Reply:
@andreyaksenov

Reply:
I vote for `wal_retention_period`. Also found similar names in other docs.